<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="admin_label">
   <property name="geometry">
    <rect>
     <x>-110</x>
     <y>10</y>
     <width>881</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Administrator Window</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>641</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="ModifyStadiums">
    <attribute name="title">
     <string>Modify Stadiums</string>
    </attribute>
    <widget class="QLabel" name="teamName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>New Team Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newTeam_field">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="stadiumName_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>New Stadium Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stadiumName_field">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="seatingCapacity_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Seating Capacity</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="seatingCapacity">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="stadiumLocation_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadium Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stadiumLocation_field">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="playingSurface_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Playing Surface</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="playingSurface_field">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>290</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="teamLeague_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Team League</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>National</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>American</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="dateOpened_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>130</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Opened</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dateOpened_field">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>150</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="distanceCenter_label">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>180</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance to Center Field</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="distance_field">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>200</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="stadiumTopology_label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>230</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadium Topology</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="topology_field">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>250</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="stadiumRooftype_label">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>280</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadium Rooftype</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>300</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ManageSouvenirs">
    <attribute name="title">
     <string>Manage Souvenirs</string>
    </attribute>
    <widget class="QTableView" name="souvenir_tableView">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>581</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_stadiumName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadium Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_souvenirName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Souvenir Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_souvenirPrice">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Souvenir Price</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_souvenirName">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_updateSouvenir">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_removeSouvenir">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_addSouvenir">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>451</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Souvenir</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_cost">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_updateSouvenir_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Souvenir</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Admin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Admin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
